dotnet/src/Connectors/Connectors.OpenAI/Helpers/OpenAIChatResponseFormatHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text;
+using System.Text.Json;
+using JsonSchemaMapper;
+using OpenAI.Chat;
+
+namespace Microsoft.SemanticKernel.Connectors.OpenAI;
+
+/// <summary>
+/// Helper class to process <see cref="ChatResponseFormat"/> object.
+/// </summary>
+internal static class OpenAIChatResponseFormatHelper
```

Comment:
Helper folders and classes tend to become a bag of random stuff over time, since they are often not well defined enough to constrain what's put into them, so they just accrue more and more stuff.
Consider a more constrained name, such as OpenAIChatResponseFormatBuilder, if this is about building ChatResponseFormat objects, so that it's a bit more constrained.